### PR TITLE
[Merged by Bors] - chore: minimize some imports

### PR DESCRIPTION
**Pull request by grunweg**
Started from Algebra/Periodic.lean with some snowball sampling. Seems to be somewhat disjoint from the tree shaking in #9347.

### CODE REVIEW

File: Mathlib/Algebra/Periodic.lean [hunk 1]
```diff
@@ -3,14 +3,11 @@ Copyright (c) 2021 Benjamin Davidson. All rights reserved.
...
 -/
-import Mathlib.Algebra.BigOperators.Basic
 import Mathlib.Algebra.Field.Opposite
-import Mathlib.Algebra.Module.Basic
 import Mathlib.Algebra.Order.Archimedean
-import Mathlib.Data.Int.Parity
 import Mathlib.GroupTheory.Coset
-import Mathlib.GroupTheory.Subgroup.ZPowers
 import Mathlib.GroupTheory.Submonoid.Membership
+import Mathlib.GroupTheory.Subgroup.ZPowers
```
Comment (jcommelin) [inline Mathlib/Algebra/Periodic.lean:10]: Why did you move this down a line? I think the list used to be alphabetically ordered... for whatever that's worth.

**Review by jcommelin — APPROVED** [review]
bors d+

File: Mathlib/Algebra/Periodic.lean (hunk 1 above)
Comment (grunweg) [reply Mathlib/Algebra/Periodic.lean:9]: Because it was late and I thought it'd be ordered this way? I don't remember; fixed.

**Comment by grunweg** [comment]
Thank you for the fast review.
bors r+
